applications/conga-cards/src/app.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright contributors to the Hyperledgendary Full Stack Asset Transfer Guide project
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
suggestion - include the stack trace  mapping logic - check the runhfsc and weft projects for examples. 
It means that any stack trace will be reported with .ts file line numbers rather than the .js line numbers

Reply:
super.   It was bugging me that the exception line numbers were always wrong.   Now they are correct.